src/asammdf/gui/widgets/tabular_base.py

```diff
@@ -1379,30 +1379,20 @@ def apply_filters(self, event=None):
             column_name = pandas_query_compatible(column_name)
             op = filter.op.currentText()
 
-            if target != target:
-                # here we have NaN
-                nan = np.nan
```

Comment:
The pandas query check the local scope, so this has to remain otherwise `@nan` will fail

Reply:
Fixed